aio/deploy/helm-chart/kubernetes-dashboard/values.yaml

```diff
@@ -98,6 +98,9 @@ service:
   ## set a specific load balancer Ip if you're using one
   # loadBalancerIP:
 
+  ## enable or disable the kubernetes.io/cluster-service label
+  clusterServiceLabel: "kubernetes.io/cluster-service: true"
```

Comment:
What should be the value for gke? Maybe it is worth mentionning it. I even wonder if this should be a boolean.

Reply:
I wasn't sure about that. My plan for my own deployments to GKE was to set clusterServiceLabel to an empty string, thereby not adding a label at all. But maybe it should be something like:
`clusterServiceLabel: 
  enabled: true
  key: "kubernetes.io/cluster-service"` 
What do you think of that?

Reply:
also, the service manifest would then contain this templated label:
`{{ .Values.service.clusterServiceLabel.key | indent 4}}: {{ .Values.service.clusterServiceLabel.enabled | quote }}`